CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ file(GLOB_RECURSE HEADERS torchvision/csrc/vision.h)
 file(GLOB_RECURSE MODELS_HEADERS torchvision/csrc/models/*.h)
 file(GLOB_RECURSE MODELS_SOURCES torchvision/csrc/models/*.h torchvision/csrc/models/*.cpp)
 
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
Well, I have to say that enabling `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` is not the best option that we should take. Because MSVC has a hard limit on the numbers of the exported symbols, which is 65535. It may be used temporarily as a workground, but I don't think we should get this in for the following reasons:
1. There is no CI test on Windows, so it will easily get broken at some time. (And at that time, it will be hard to fix.)
2. The c++ codebase in this repo is not so big. So adding annotation is possible.

Reply:
Here is how we fix that in pytorch/pytorch: https://github.com/pytorch/pytorch/pull/9491.

Reply:
Ah no problem, makes sense. Is there any preference on the naming of the macro, something like

`#define TV_CPP_API`

since the lib being built is called torchvision?

Reply:
@philipNoonan Maybe `VISION_API` is a good option.